sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```diff
@@ -550,1735 +534,1713 @@ dmnFullTimeIncomeThreshold</bpmn:text>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0y5azw6</bpmn:incoming>
-      <bpmn:outgoing>Flow_0z2payo</bpmn:outgoing>
+      <bpmn:incoming>Flow_14thi6n</bpmn:incoming>
+      <bpmn:outgoing>Flow_0dqxcli</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0fjl0zr" name="BCSL After Top Up">
+    <bpmn:businessRuleTask id="Activity_1wciqrl" name="dmnFullTimeApplicationBooksMaximum">
+      <bpmn:documentation>The application limit for books and supplies based on </bpmn:documentation>
+      <bpmn:extensionElements>
+        <zeebe:calledDecision decisionId="dmnFullTimeApplicationBooksMaximum" resultVariable="dmnFullTimeApplicationBooksMaximum" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_10egxdo</bpmn:incoming>
+      <bpmn:outgoing>Flow_15ld0wf</bpmn:outgoing>
+    </bpmn:businessRuleTask>
+    <bpmn:intermediateThrowEvent id="Event_1kr3rjl" name="Program Year Award Total Initialization">
+      <bpmn:documentation>CSGP (CSG-D) is FT/PT
+SBSD is FT/PT
+BGPD (BCAG-D) has a min award that it could be rounded up to so weeks are insufficient to determine if max per year
+BCAG has a min award that it could be rounded up to so weeks are insufficient to determine if max per year
+
+</bpmn:documentation>
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=(calculatedDataTotalCSG+federalAwardNetCSLFAmount+provincialAwardNetBCSLAmount)" target="calculatedDataPrelimAward" />
-          <zeebe:output source="=//remaining need and negative amounts will reduce BCSL&#10;calculatedDataProvincialAssessedNeed-calculatedDataPrelimAward" target="provincialAwardUnmetNeedAfterLoanAndCSG" />
-          <zeebe:output source="=max(calculatedDataBCTopup-calculatedDataPrelimAward,0)" target="calculatedDataTopup" />
-          <zeebe:output source="=min(calculatedDataNetTopup,provincialAwardUnmetNeedAfterLoanAndCSG)" target="calculatedDataNetTopup" />
+          <zeebe:output source="=if(programYearTotalFullTimeCSGP = null) then 0 else programYearTotalFullTimeCSGP" target="programYearTotalFullTimeCSGP" />
+          <zeebe:output source="=if(programYearTotalPartTimeCSGP = null) then 0 else programYearTotalPartTimeCSGP" target="programYearTotalPartTimeCSGP" />
+          <zeebe:output source="=if(programYearTotalFullTimeSBSD = null) then 0 else programYearTotalFullTimeSBSD" target="programYearTotalFullTimeSBSD" />
+          <zeebe:output source="=if(programYearTotalPartTimeSBSD = null) then 0 else programYearTotalPartTimeSBSD" target="programYearTotalPartTimeSBSD" />
+          <zeebe:output source="=programYearTotalFullTimeSBSD + programYearTotalPartTimeSBSD" target="programYearTotalSBSD" />
+          <zeebe:output source="=programYearTotalFullTimeCSGP + programYearTotalPartTimeCSGP" target="programYearTotalCSGP" />
+          <zeebe:output source="=if(programYearTotalBGPD = null) then 0 else programYearTotalBGPD" target="programYearTotalBGPD" />
```

Comment:
`programYearTotalBGPD` is this a placeholder for something to be implemented in future?